app.py

```diff
@@ -66,11 +78,14 @@ def gradient_temp(temp, ideal):
 
 @app.template_filter("day")
 def day(value, format="%A %d %b"):
```

Comment:
```suggestion
def day(value):
```

Looks like we don't need that anymore.